manifests/init.pp

```diff
@@ -155,18 +156,19 @@
 
   include foreman_proxy_content::pub_dir
 
-  if $qpid_router {
-    class { 'foreman_proxy_content::dispatch_router':
-      agent_addr    => $qpid_router_agent_addr,
-      agent_port    => $qpid_router_agent_port,
-      ssl_ciphers   => $qpid_router_ssl_ciphers,
-      ssl_protocols => $qpid_router_ssl_protocols,
-      logging_level => $qpid_router_logging_level,
-      logging       => $qpid_router_logging,
-      logging_path  => $qpid_router_logging_path,
-    }
-    contain foreman_proxy_content::dispatch_router
+  class { 'foreman_proxy_content::dispatch_router':
```

Comment:
My concern hasn't been addressed: previously it was possible to not even install qpid router at all. Now it's always installed and configured, just disabled.

Perhaps we need to implement this:
```suggestion
  class { 'foreman_proxy_content::dispatch_router':
    ensure => bool2str($enable_katello_agent, 'present', 'absent'),
```

Reply:
Take a look at https://github.com/theforeman/puppet-qpid/pull/154